micropython/micropython-1.15-patchset/py.patch

```diff
@@ -1,50 +1,65 @@
+diff -Naur micropython-1.15/py/emitnative.c micropython-phoenix/py/emitnative.c
+--- micropython-1.15/py/emitnative.c	2021-04-18 16:27:07.000000000 +0200
++++ micropython-phoenix/py/emitnative.c	2021-05-25 09:43:17.351537610 +0200
+@@ -2396,7 +2396,7 @@
+         } else if (op == MP_BINARY_OP_MULTIPLY) {
+             ASM_MUL_REG_REG(emit->as, REG_ARG_2, reg_rhs);
+             emit_post_push_reg(emit, vtype_lhs, REG_ARG_2);
+-        } else if (MP_BINARY_OP_LESS <= op && op <= MP_BINARY_OP_NOT_EQUAL) {
++        } else if (MP_BINARY_OP_LESS == op && op <= MP_BINARY_OP_NOT_EQUAL) {
```

Comment:
would be great if You could add some comment with explanation why this is needed (eg. `// PHOENIX CHANGE: needed because "-3 < 5" fails on our system due to singbit() invalid implementation`)

Reply:
Comment added where i think it was necessary